README.md

```diff
@@ -77,21 +77,15 @@ psql -h localhost
 
 #### Node Dependencies
 
-Install nvm (node version manager):
-
-```
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.1/install.sh | bash
-```
-__Note:__ additional nvm installation notes found at [the nvm repo](https://github.com/creationix/nvm#install-script).
```

Comment:
https://github.com/asdf-vm/asdf-nodejs has a few other dependencies, so we might want to keep a note regarding more installation notes at their repo

Reply:
Noted in the new amended commit but its not making this diff show outdated.